.github/workflows/build-capi-vm-images.yml

```
@@ -0,0 +1,38 @@
+name: Build CAPI VM image with manual input
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: k8s version
+        required: true
+        type: string
+        default: "1.33.1"
+      tag:
+        description: ck8s capi version
+        required: true
+        type: string
+        default: "0.8"
+      builder_image:
+        description: image builder image
+        required: true
+        type: string
+        default: "ghcr.io/elastisys/image-builder-amd64:main"
+
+env:
+  PACKER_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build-azure-image:
+    uses: ./.github/workflows/build-azure-capi-image.yml
+    with:
+      version: ${{ inputs.version || '1.33.1' }}
```

Question: Do we need the defaults in 2 places? Same on the tag etc.

```suggestion
      version: ${{ inputs.version }}
```

---

If i remember correctly, yes, as the default was not always picked up.

---

Hmm that feels odd, it like the inputs section would be enough. Could it be that it's marked as `required` and has a `default` value? Not sure why that should mess with the defaults, but that's the only thing that I can see as a bit inconsistent